packages/wrangler/src/versions/upload.ts

```diff
@@ -677,7 +677,12 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 		// Upload assets if assets is being used
 		const assetsJwt =
 			props.assetsOptions && !props.dryRun
-				? await syncAssets(accountId, scriptName, props.assetsOptions.directory)
+				? await syncAssets(
+						accountId,
+						props.assetsOptions.directory,
+						scriptName,
+						undefined
```

Comment:
nit: should we just leave it out instead of passing `undefined`?

Reply:
Can you tell I've been writing a lot of Go recently?

Reply:
lol